package.json

```diff
@@ -83,13 +83,13 @@
     "redux-mock-store": "^1.2.3",
     "redux-throttle": "0.1.1",
     "rimraf": "^2.6.1",
-    "scratch-audio": "latest",
-    "scratch-blocks": "latest",
-    "scratch-l10n": "latest",
-    "scratch-paint": "latest",
-    "scratch-render": "latest",
-    "scratch-storage": "^0.3.0",
-    "scratch-vm": "latest",
+    "scratch-audio": "0.1.0-prerelease.1511362862",
+    "scratch-blocks": "0.1.0-prerelease.1513170598",
+    "scratch-l10n": "2.0.20171211145754",
```

Comment:
Since eventually this version will get bumped periodically, every time there are new translations, we might want this to be patch or minor-pinned. Let's leave it for now though.

Reply:
Yeah we'll have to decide what exactly the versioning scheme is for language updates. are all language updates patch changes? Are new languages major/minor/patch? I assume major versions would still only be for things like package structure changes? Semver may or may not apply quite right to those ideas.